src/servant/tt_001/F84.c

```diff
@@ -6,7 +6,68 @@ extern s32 g_IsServantDestroyed;
 
 #include "../set_entity_animation.h"
 
-INCLUDE_ASM("servant/tt_001/nonmatchings/F84", func_us_80170F84);
+s32 UpdateEntityVelocityTowardsTarget(s32 unused, s32 targetX, s32 targetY) {
```

Comment:
There's an unused `arg0` here - maybe a hint that this code is common elsewhere and it is used there